ui/src/main/resources/Meeting/Code/WebHomeSheet.xml

```diff
@@ -60,11 +61,20 @@ $services.localization.render('contrib.meeting.home.presentation', ['[[**', '**&
   'translationPrefix': 'meeting.livetable.',
   'tagCloud': true,
   'rowCount': 15,
-  'maxPages': 10,
-  'selectedColumn': 'startDate',
-  'defaultOrder': 'desc'
+  'maxPages': 10
 })
-#set ($columns = ['doc.title', 'leader', 'place', 'startDate', 'durationDisplay', 'status', '_actions'])
+#if ($request.getParameter("meeting") == 'all')
```

Comment:
I think that you should use single quotes for `'meeting'` as well, like you did for `'all'`